routemaster-drain.gemspec

```diff
@@ -18,10 +18,10 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency     'faraday', '>= 0.9.0'
   spec.add_runtime_dependency     'faraday_middleware'
-  spec.add_runtime_dependency     'net-http-persistent', '< 3' # 3.x is currently incompatible with faraday
+  spec.add_runtime_dependency     'typhoeus', '~> 1.1'
```

Comment:
For once, yes. Typhoeus uses a C library and a lot of threading, so I'd be uncomfortable with a non-patchlevel upgrade without a lot of testing.